src/cdk/testing/protractor/protractor-element.ts

```diff
@@ -73,7 +73,7 @@ export class ProtractorElement implements TestElement {
     return this.element.clear();
   }
 
-  async click(relativeX = 0, relativeY = 0): Promise<void> {
+  async click(relativeX = 1, relativeY = 1): Promise<void> {
     await browser.actions()
```

Comment:
Should we click at the center of the elementcby default? This what Protractor did before we added the mouse move. https://www.selenium.dev/selenium/docs/api/javascript/module/selenium-webdriver/lib/actions_exports_LegacyActionSequence.html#click